contributors/onboarding/staff-onboarding-guide/engineer-onboarding.md

```diff
@@ -0,0 +1,111 @@
+# Mattermost Engineer Onboarding Timeline & Expectations
```

Comment:
Great to see this surfaced in the handbook! Looks good to me.

One note is that I'm not sure this can fully /replace/ https://docs.google.com/document/d/14jsUJgGO4j5f4B6bPVCIC1y9yXy43rmAFI0F5MJsuE0/edit, since I've copied the original document for each new staff member I manager and use it as a sort of 1:1 checklist to ensure alignment. It's meant to be a sort of living document in their context as well, with Google Docs being a perfect medium for sharing comments and such.

Reply:
Thanks for raising this. My opinion is that this page is more of an outline for candidates/new staff to give an overview of expectations and process. I don't see a problem with using a Google doc like yours unless it deviates substantially from this content. I would expect each team/manager to have items that aren't mentioned in this document so a bit of customization shouldn't cause confusion. 

Totally my opinion though, @jasonblais needs to weigh in as well.

Reply:
@lieut-data I'm wondering if the same steps could be taken from the handbook -- which is to create a fresh Google Doc and copy-paste the content from the handbook. 

It does add an extra step into that workflow, but I worry the handbook page will become outdated if the Google Doc is continued to be used as the source of truth.

Reply:
(And if the Google Doc is continued to be used, then perhaps we should just link to that doc from the handbook instead)

Reply:
I'm fine with the copy/paste proposal. @jwilander, I know you wrote the original version of this very helpful article. Any thoughts?

Reply:
I like having the source of truth copy in the handbook 👍 I'm good with copy/pasting it into a google doc as needed